package.json

```diff
@@ -30,7 +30,7 @@
   "peerDependencies": {
     "bedrock": "^1.0.6",
     "bedrock-docs": "^2.0.0",
-    "bedrock-identity": "^4.1.0",
+    "bedrock-identity": "digitalbazaar/bedrock-identity#modelFields",
```

Comment:
Remember to change to new semver minor version of bedrock-identity once released.